example/eblob_backend.cpp

```diff
@@ -487,6 +487,23 @@ static int blob_read_new_impl(eblob_backend_config *c,
 		wc.size -= sizeof(ehdr) + ehdr.size;
 		wc.data_offset += sizeof(ehdr) + ehdr.size;
 		record_offset += sizeof(ehdr) + ehdr.size;
+
+		auto data_size = wc.size;
+		auto data_offset = wc.data_offset;
+
+		if (request.read_flags & DNET_READ_FLAGS_DATA) {
+			// TODO: should be checked `jhdr.capacity > data_size` case?
+			data_size -= jhdr.capacity;
```

Comment:
Why are `data_size` and `data_offset` shifted when `if (request.read_flags & DNET_READ_FLAGS_DATA) {}`?

Reply:
WIP artifact, fixed.